File: 백준/Bronze/18406. 럭키 스트레이트/럭키 스트레이트.py
n = input()
sum_left = 0
sum_right = 0

for i in range(len(n) // 2):
    sum_left += int(n[i])
for j in range(len(n) // 2, len(n)):
    sum_right += int(n[j])

if sum_left == sum_right:
    print("LUCKY")
else:
    print("READY")

File: 백준/Bronze/18406. 럭키 스트레이트/럭키 스트레이트.java
import java.util.Scanner;

public class Main {
    static char[] n;
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        n = sc.next().toCharArray();
        int len = n.length;

        int sum1 = 0;
        int sum2 = 0;

        for(int i=0; i<len/2; i++){
            sum1 += n[i] - '0';
        }
        for(int i=len/2; i<n.length; i++){
            sum2 += n[i] - '0';
        }

        if (sum1 == sum2)
            System.out.println("LUCKY");
        else
            System.out.println("READY");
    }
}
